MVCapplication/src/main/java/com/softserve/if072/mvcapp/controller/UserProfileController.java

```diff
@@ -0,0 +1,48 @@
+package com.softserve.if072.mvcapp.controller;
+
+import com.softserve.if072.common.model.User;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Arrays;
+
+/**
+ * This class provides access to the user's profile page.
+ *
+ * @author Oleh Pochernin
+ */
+@Controller
+public class UserProfileController {
+    private static final String GET_CURRENT_USER = "http://localhost:8080/rest/api/security/user/current";
```

Comment:
Guys, please move all URL to property file. And in property extrach http://localhost:8080 from the rest url, for better maintaiablity, for example:

`app.url=http://localhost:8080`
`service.user.current=${app.url}/rest/api/security/user/current`